src/table/components/TableBodyWrapper.tsx

```diff
@@ -76,6 +76,8 @@ function TableBodyWrapper({
         >
           {columns.map((column, columnIndex) => {
             const { id, align } = column;
+            // Note that rows are not necessarily ordered in qColumnOrder.
+            // So for each row, the cells are mapped according to column.id
```

Comment:
Not sure we want it this way. I am considering changing to the `rows` array being ordered in the way we show things. And then just keep track of the way they are stored in the back-end with the index stored on the cell. I think this is left from your prototype @Caele. If so, do you remember if there is any particular reason to keep columns in the current order?